static/css/restyle.less

```diff
@@ -672,8 +674,9 @@ button.good {
 }
 
 .install-button > .button.add.installer {
+    box-shadow: 0 0 4px rgba(0, 0, 0, 0.4);
+
     background: #57BD35;
-    box-shadow: 0 2px 0 0 rgba(0,0,0,0.10), inset 0 -2px 0 0 rgba(56,136,30,0.50);
```

Comment:
Does this bit need to be changed too? It doesn't look to be related to the overlay.